NDB.Covid19/NDB.Covid19/Configuration/OAuthConf.cs

```diff
@@ -2,11 +2,12 @@ namespace NDB.Covid19.Configuration
 {
     public static class OAuthConf
     {
-        public static string OAUTH2_CLIENT_ID = "INJECTED_IN_APP_CENTER_DURING_BUILD";
+        public static string OAUTH2_CLIENT_ID = "smittestopp";
```

Comment:
Why has this been changed?

Reply:
I wanted to minimize the setup when running the app from a local dev environment. Especially values that must be changed locally and thus risk being committed. This value will always be "smittestopp", so it might as well be in the source code (i believe the script will still replace it)